public/uploads/_/originals/.htaccess

```diff
@@ -8,6 +8,12 @@
     deny from all
 </FilesMatch>
 
+<FilesMatch "\.(jpe?g|png|gif|mp3|wav|ogg|m4a|mp4|mov|wmv|avi)$">
```

Comment:
Please add  `<IfModule mod_headers.c>` for those who have not enabled `mod_headers` module of Apache. Otherwise, it will throw an error.